src/sentry/migrations/0028_discover_query_upgrade.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# Generated by Hand
+from __future__ import unicode_literals, print_function
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+from sentry.discover.models import DiscoverSavedQuery, DiscoverSavedQueryProject
+from sentry.api.event_search import SearchVisitor
+from django.db import migrations
+
+# Aggregates are now fields
+def convert_field(fieldname, unique, reverse):
+    if fieldname == "count":
+        fieldname = u"count()"
+    elif unique:
+        fieldname = u"count_unique({})".format(fieldname) if unique else field
+
+    fieldname = u"-{}".format(fieldname) if reverse else fieldname
+    return fieldname
+
+def prepare_value(value):
+    value = value.replace("%", "*")
+    if " " in value and not value.startswith('"'):
+        value = u'"{}"'.format(value)
+    return value
+
+def convert(saved_query, name_extra=" (migrated from legacy discover)"):
```

Comment:
I added an extra bit to the end of each name to make it clearer where these new queries are coming from: 
![image](https://user-images.githubusercontent.com/4205004/73395654-f119d980-42ad-11ea-9526-49cb0b85f69e.png)

Reply:
Nice touch :heart: